deploy/helm/kubecf/templates/bosh_deployment.yaml

```diff
@@ -16,10 +16,10 @@ spec:
     name: cf-deployment
     type: configmap
   ops:
-  - name: {{ include "kubecf.ops-name" (dict "Path" "assets/operations/unordered/move_auctioneer.yaml") }}
-    type: configmap
-  - name: {{ include "kubecf.ops-name" (dict "Path" "assets/operations/unordered/move_routing_api.yaml") }}
+{{- range $path, $bytes := .Files.Glob "assets/operations/job_moving/*" }}
```

Comment:
We never use `$bytes` here.

Reply:
Fixed in https://github.com/cloudfoundry-incubator/kubecf/pull/619/commits/49c5c179b823834d7b592b1d83725f079dc3312f.